Options.py

```diff
@@ -280,14 +281,19 @@ def verify_keys(cls, data):
                                 f"Allowed keys: {cls.valid_keys}.")
 
     def verify(self, world):
+        if self.convert_name_groups and self.verify_item_name:
```

Comment:
> and self.verify_item_name:

No verification is happening in this branch, right? Is this key necessary?

Reply:
verify_item_name tells us the contents is actually items, not something else